resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -552,6 +567,9 @@ export default {
                     .finally(this.finishLoading);
             }
         },
+        cachedImages() {
+            this.cachePreviousAndNext();
```

Comment:
This could make use of the `debounce` function so the backend is not spammed with requests for images while the user interacts with the slider. `debounce` waits for a specific time until it is no longer called before a specified callback function is executed. I set 100 ms here but maybe we could even increase that to 500 ms.

```suggestion
            debounce(this.cachePreviousAndNext, 100, 'annotations.cached-image-count.update');
```

Reply:
I just copy and paste your suggestion